lib/fog/vsphere/compute.rb

```diff
@@ -284,6 +287,24 @@ def parent_attribute(path, type)
           nil
         end
 
+        # Maps RbVmomi values to understandable strings.
+        # Result is uniqued, what effectively means that the order is defined by the first occurence.
+        def parse_boot_order(values)
+          return unless values.is_a?(Array)
+          values.map do |v|
```

Comment:
nit: can we please call this variable `value`? Thanks.

Reply:
I have renamed it to `vm_boot_dev`(s) be more descriptive. Sounds better?